forge-game/src/main/java/forge/game/Game.java

```diff
@@ -265,6 +265,7 @@ public Game(Iterable<RegisteredPlayer> players0, GameRules rules0, Match match0,
         for (RegisteredPlayer psc : players0) {
             IGameEntitiesFactory factory = (IGameEntitiesFactory)psc.getPlayer();
             Player pl = factory.createIngamePlayer(this, plId++);
+            pl.setHasLivingEnd(psc.getDeck().getMain().countByName("Living End", true) > 0);
```

Comment:
Should probably be part of AiController instead

Reply:
Agreed